src/main/java/pl/marcin/project/RunnerFactory.java

```diff
@@ -0,0 +1,31 @@
+package pl.marcin.project;
+
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import pl.marcin.project.runners.AppRunner;
+import pl.marcin.project.runners.ShopAppRunner;
+import pl.marcin.project.runners.WebAppRunner;
+
+import java.util.Scanner;
+
+
+public class RunnerFactory implements ApplicationRunner {
+
+    private AppRunner appRunner;
+
+    @Override
+    public void run(ApplicationArguments args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("How do you want to run your application: web or shop ?");
+
+        String runningWay = scanner.nextLine();
+        if ("web".equals(runningWay)) {
+            appRunner = new WebAppRunner();
+        } else if ("shop".equals(runningWay)) {
```

Comment:
please handle the case where the user doesn't provide one of the accepted values :)

<img width="1419" alt="image" src="https://github.com/marcingas/project/assets/78318543/b3f53c80-3dbe-46ab-a0c3-ed6bf15ce8ed">